catalog/src/main/java/org/polypheny/db/catalog/CatalogImpl.java

```diff
@@ -1543,10 +1539,10 @@ public long addMaterializedView( String name, long schemaId, int ownerId, TableT
         CatalogSchema schema = getSchema( schemaId );
         CatalogUser owner = getUser( ownerId );
 
-        //Technically every Table is partitioned. But tables classified as UNPARTITIONED only consist of one PartitionGroup and one large partition
+        // Technically every Table is partitioned. But tables classified as UNPARTITIONED only consist of one PartitionGroup and one large partition
         List<Long> partitionGroupIds = new ArrayList<>();
         partitionGroupIds.add( addPartitionGroup( id, "full", schemaId, PartitionType.NONE, 1, new ArrayList<>(), true ) );
-        //get All(only one) PartitionGroups and then get all partitionIds  for each PG and add them to completeList of partitionIds
+        // get All(only one) PartitionGroups and then get all partitionIds  for each PG and add them to completeList of partitionIds
```

Comment:
This comment is hard to comprehend...

Reply:
No, I only added the space. The original comment is from @isabelge

Reply:
I don't think so, and git blames @hennlo😉

Reply:
Yes, you are right. It has been introduced in this commit https://github.com/polypheny/Polypheny-DB/commit/ea206c9d4a64a76cded8968ebf90d35cac2c21c3 by @hennlo

Reply:
Oh wow this nested comment is almost artistical. I'll revamp it 👍

Reply:
Fixed in 6b11bed